backend/app/services/internal/file_upload.py

```
@@ -0,0 +1,47 @@
+from typing import Optional
+from app.core.exceptions.http_exception import BadRequestException
+from app.services.repositories.user_file_repository import UserFileRepository
+from app.core.exceptions.http_exception import NotFoundException, BadRequestException
```

Ruff (0.14.10)

Redefinition of unused `BadRequestException` from line 2

Remove definition: `BadRequestException`

(F811)